kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -62,6 +77,16 @@
         default: false,
       },
     },
+    data() {
+      return {
+        hasTooltip: false,
+      };
+    },
+    updated() {
```

Comment:
I believe that this only gets triggered when vue's updates the dom, but not when the dom is resized.

In order to handle resize events, we can probably leverage our "responsive element" mixin.

Reply:
> I believe that this only gets triggered when vue's updates the dom, but not when the dom is resized.

Yeah, I was looking for other alternatives in Vue's documentation but couldn't find an event that fires when the dom is resized.